### ISSUE TITLE
Peek does not produce any output under Ubuntu 18.04 bionic ("Missing output file after post processing.")

### ISSUE THREAD

**Issue by bentolor** [phw/peek#363]
Under a current 18.04 installation I'm failing to use this lovely tool at all. In short

* GIF and APNG lead to an error `Missing output file after post processing.`
* All other variants simply produce a 0-byte long, empty file

I assume I'm reporting here a more vebose duplicate of #360 and #343 

Starting via command line does not reveal that much additional info. And I did not find a `--debug` switch.

Can I support in diagnosing this problem?

**Comment by bentolor** [comment]
As a workaround: Uninstalling via PPA (`sudo apt purge peek` and then installing via: `flatpak install flathub org.gnome.Platform//3.26` and `flatpak install --user https://flathub.org/repo/appstream/com.uploadedlobster.peek.flatpakref` produces a version which works fine.

**Comment by phw** [comment]
I assume you were missing some dependencies. Are you recording on Gnome? X or Wayland? Are ffmpeg and the GStreamer good and ugly plugins installed?

**Comment by bentolor** [comment]
I have vanilla `ubuntu-desktop` installed as well as `ffmpeg`, `libgstreamer1.0-0` including `libgstreamer-plugins-good1.0-0`, `libgstreamer-plugins-base1.0-0`, `libgstreamer-plugins-bad1.0-0` and `libgstreamer-gl1.0-0`.

I'm using the default Ubuntu desktop which means Gnome Shell with X (?).

**Comment by phw** [comment]
Thanks for the details. Can you have `journalctl -f` running while doing a Peek recording and see if there is any output related to this? Probably something by gnome-shell?

Also please test the following:

1. Please run `loginctl show-session $(loginctl | grep $(whoami) | awk '{print $1}') -p Type` to check your session type. It will either output `Type=x11` or `Type=wayland` (I think the default on Ubuntu should be X, but not sure)
2. See if  the built-in Gnome screen recorder works. By default I think you start and stop recording the screen with Shift+Ctrl+Alt+r, the result is placed in your Videos folder

A workaround, at least if you are on X, could be to start Peek with `peek -b ffmpeg`. This will use ffmpeg for screen recording and should work as long as you are not using Wayland.

**Comment by bentolor** [comment]
Hello @phw 

TL;DR: For some reasons Peek started to work. Typical user-claim™ provided:  _I did not change anything!1!_

For the records: 

1. `Type=x11`
2. Gnome screen recorder works, puts them in my home directory (`Video` is a symblink to a write-protected location)
3. `journalctf -f` does not provide any errors related to  this issue.

Sorry for bothering you. And thanks for your help! 
If I'm able to reproduce, I'll come back to this issue.
